+opencossan/+bayesiannetworks/@CredalNetwork/compute_conditionals.m

```diff
@@ -0,0 +1,112 @@
+function [prob_low, prob_hi] = compute_conditionals(parent_states, parent_data, node_states, node_data, conf)
+    %%%
+    %   Given a call array of states, a call array of data (samples), node states,
```

Comment:
I'm assuming there is no way around the cell arrays because the dimensions aren't the same?

Reply:
Hmm, I think this is the best way to do it.. when I first implemented it I had to hard code lots of `for` loops. I think this is the best way to have a variable number of parents for the function. 

You don't know the number of parents, or how much states they each have (which could be all different) ...